tinygrad/runtime/ops_torch.py

```diff
@@ -27,4 +27,4 @@ def fromCPU(cls, x):
     buf = torch.from_numpy(x).requires_grad_(False).to(device)
     return cls(prod(x.shape), type_map[buf.dtype], buf)
   def toCPU(self): return self._buf.cpu().numpy()
-TorchBuffer = Interpreted(RawTorchBuffer, torch_fxn_for_op, from_underlying=lambda x: RawTorchBuffer(prod(x.shape), type_map[x.dtype], x))
+TorchBuffer = Interpreted(RawTorchBuffer, torch_fxn_for_op, from_underlying=lambda x: RawTorchBuffer(prod(x.shape), type_map[x.dtype], x),to_underlying=lambda x: x._buf if x._buf.dtype!=torch.bool else x._buf.long(), cast_underlying=lambda x,dtype: x.to(inverse_type_map[dtype])) #Torch has a different type-casting logic (e.g. sqrt of int32 is float32), fixing this with cast_underlying.
```

Comment:
needs a space between `x),` and `to_underlying`

Reply:
done